### PR TITLE
added a note and fixed flaws

### LINKED ISSUE #1644: Issue with "<easing-function>": unclear that `steps` applies to each segment of the animation
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/easing-function

#### What information was incorrect, unhelpful, or incomplete?
The documentation doesn't mention that `steps`, unlike the other possible values of `<easing-function>`, applies to each segment of the keyframes animation, rather than to its entire duration.

#### Specific section or headline?

_The steps() class of easing functions_

Note that this information (or a subset of it is also included in [animation-timing-function](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-timing-function)

#### What did you expect to see?

Some mention of this behavior when there's multiple stops in the animation.

#### Did you test this? If so, how?

```html
<div class="animation-1">steps(1, end)</div>
<div class="animation-3">steps(3, end)</div>
```

```css
div {
  font-weight: 700;
  animation: rgb 10s infinite;
}

div.animation-1 {
  animation-timing-function: steps(1, end);
}

div.animation-3 {
  animation-timing-function: steps(3, end);
}

@keyframes rgb{
  0% {
    color: #f00;
  }
  33.33% {
    color: #0f0;
  }
  66.66% {
    color: #00f;
  }
}
```

[see live demo](https://steps-example.glitch.me)

I was expecting `steps(3, end)` to use the 0% value (red) for a third of the animation duration, then the 33.33% value (green) for another third, and then the 66.67% value (blue) for the remaining third.

Instead `steps(3, end)` split _each segment_ of the animation into three parts (so 3 * 3 = 9 in total). In order to achieve my desired animation I had to use `steps(1, end)`.

https://user-images.githubusercontent.com/777023/105553700-8e1b2100-5d0e-11eb-9158-aff991d6e2d7.mov

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/easing-function`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/easing-function
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/css/easing-function/index.html
* Document last modified: 2021-01-09T13:41:23.000Z

</details>

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes #1644

Added a note about multiple stop animations, also fixed the fixable flaws with images while in there.

### CODE REVIEW

File: files/en-us/web/css/easing-function/index.html [hunk 1]
```diff
@@ -129,26 +129,26 @@ <h5 id="steps_n_&lt;direction&gt;">steps( n, &lt;direction&gt; )</h5>
  <li>
   <p><code>steps(2, jump-start)</code><br>
    <code>steps(2, start)</code></p>
-  <img src="/files/3436/steps(2,start).png" style="float: left; height: 327px; vertical-align: top;"></li>
+  <img src="steps(2,start).png" style="float: left; vertical-align: top;"></li>
```
Comment (ericwbailey) [inline files/en-us/web/css/easing-function/index.html:132]:
```suggestion
  <img alt="A graph with X and Y ranges from 0 to 1, with the X axis labeled 'Time ratio' and the Y axis labeled 'Output ratio.' Three dots are present, the first at the X 0 Y 0 position, the second at the X 0.5 Y 0.5 position, and the third at the X 1 Y 1 position. The second and third dots have horizontal lines extending 0.5 units backwards towards the Y axis." src="steps(2,start).png" style="float: left; vertical-align: top;"></li>
```

File: files/en-us/web/css/easing-function/index.html [hunk 2]
```diff
@@ -129,26 +129,26 @@ <h5 id="steps_n_&lt;direction&gt;">steps( n, &lt;direction&gt; )</h5>
...
  <li>
   <p><code>steps(4, jump-end)<br>
    steps(4, end)</code></p>
-  <img alt="four steps, with a jump from the fourth step to the final value at the 100% mark" src="/files/3437/steps(4,end).png"></li>
+  <img alt="four steps, with a jump from the fourth step to the final value at the 100% mark" src="steps(4,end).png"></li>
```
Comment (ericwbailey) [inline files/en-us/web/css/easing-function/index.html:136]:
```suggestion
  <img alt="Four steps, with a jump from the fourth step to the final value at the 100% mark." src="steps(4,end).png"></li>
```

File: files/en-us/web/css/easing-function/index.html [hunk 3]
```diff
@@ -129,26 +129,26 @@ <h5 id="steps_n_&lt;direction&gt;">steps( n, &lt;direction&gt; )</h5>
...
  <li>
   <p><code>steps(5, jump-none)</code></p>
-  <img alt="five steps, with no jumps, so 20% of the time is at the beginning state or 0% mark and the last 20% is at the final state, or 100% mark" src="/files/16419/step5none.png"></li>
+  <img alt="five steps, with no jumps, so 20% of the time is at the beginning state or 0% mark and the last 20% is at the final state, or 100% mark" src="step5none.png"></li>
```
Comment (ericwbailey) [inline files/en-us/web/css/easing-function/index.html:139]:
```suggestion
  <img alt="Five steps, with no jumps, so 20% of the time is at the beginning state or 0% mark and the last 20% is at the final state, or 100% mark." src="step5none.png"></li>
```

File: files/en-us/web/css/easing-function/index.html [hunk 4]
```diff
@@ -129,26 +129,26 @@ <h5 id="steps_n_&lt;direction&gt;">steps( n, &lt;direction&gt; )</h5>
...
  <li>
   <p><code>steps(3, jump-both)</code></p>
-  <img src="https://mdn.mozillademos.org/files/16420/step3both.png"></li>
+  <img src="step3both.png"></li>
```
Comment (ericwbailey) [inline files/en-us/web/css/easing-function/index.html:142]:
```suggestion
  <img alt="A graph with X and Y ranges from 0 to 1, with the X axis labeled 'Time ratio' and the Y axis labeled 'Output ratio.' Five dots are present, the first at the X 0 Y 0 position, the second at the X 0 Y 0.25 position, the third at the X 0.5 Y 0.5 position, the fourth at the X 0.75 Y 0.75 position, and the fifth at the X 1 Y 1 position. The second, third, and fourth dots have horizontal lines extending 0.25 units forwards away from the Y axis." src="step3both.png"></li>
```

File: files/en-us/web/css/easing-function/index.html [hunk 5]
```diff
@@ -129,26 +129,26 @@ <h5 id="steps_n_&lt;direction&gt;">steps( n, &lt;direction&gt; )</h5>
...
 </ul>
 
 <h5 class="cleared" id="step-start">step-start</h5>
 
-<p><img src="/files/3423/steps(1,start).png" style="float: left; height: 327px; vertical-align: top;"> The interpolation jumps immediately to its final state, where it stays until the end. This keyword represents the easing function <code>steps(1, jump-start)</code> or <code>steps(1, start)</code>.</p>
+<p><img src="steps(1,start).png" style="float: left; vertical-align: top;"> The interpolation jumps immediately to its final state, where it stays until the end. This keyword represents the easing function <code>steps(1, jump-start)</code> or <code>steps(1, start)</code>.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/easing-function/index.html:147]:
```suggestion
<p><img alt="A graph with X and Y ranges from 0 to 1, with the X axis labeled 'Time ratio' and the Y axis labeled 'Output ratio.' Two dots are present, the first at the X 0 Y 0 position, and the second at the X 1 Y 1 position. The second dot has a horizontal lines extending 1 units back towards the Y axis." src="steps(1,start).png" style="float: left; vertical-align: top;"> The interpolation jumps immediately to its final state, where it stays until the end. This keyword represents the easing function <code>steps(1, jump-start)</code> or <code>steps(1, start)</code>.</p>
```

File: files/en-us/web/css/easing-function/index.html [hunk 6]
```diff
@@ -129,26 +129,26 @@ <h5 id="steps_n_&lt;direction&gt;">steps( n, &lt;direction&gt; )</h5>
...
 
 <h5 class="cleared" id="step-end">step-end</h5>
 
-<p><img src="/files/3424/steps(1,end).png" style="float: left; height: 327px; vertical-align: top;"> The interpolation stays in its initial state until the end, at which point it jumps directly to its final state. This keyword represents the easing function <code>steps(1, jump-end)</code> or <code>steps(1, end)</code>.</p>
+<p><img src="steps(1,end).png" style="float: left; vertical-align: top;"> The interpolation stays in its initial state until the end, at which point it jumps directly to its final state. This keyword represents the easing function <code>steps(1, jump-end)</code> or <code>steps(1, end)</code>.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/easing-function/index.html:151]:
```suggestion
<p><img alt="A graph with X and Y ranges from 0 to 1, with the X axis labeled 'Time ratio' and the Y axis labeled 'Output ratio.' Two dots are present, the first at the X 0 Y 0 position, and the second at the X 1 Y 1 position. The first dot has a horizontal lines extending 1 unit forwards away from the Y axis." src="steps(1,end).png" style="float: left; vertical-align: top;"> The interpolation stays in its initial state until the end, at which point it jumps directly to its final state. This keyword represents the easing function <code>steps(1, jump-end)</code> or <code>steps(1, end)</code>.</p>
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]
Added some `alt` descriptions while I was reviewing. Back to you to review those, if you don't mind.

File: files/en-us/web/css/easing-function/index.html [hunk 7]
```diff
@@ -38,13 +38,13 @@ <h4 id="The_linear_class_of_easing_functions">The linear class of easing functio
 
 <h5 id="linear">linear</h5>
 
-<p style="float: left;"><img src="/files/3425/cubic-bezier,linear.png" style="height: 325px; width: 244px;"></p>
+<p style="float: left;"><img src="cubic-bezier,linear.png"></p>
```
[the remark this answers is not in the document]
Comment (rachelandrew) [reply files/en-us/web/css/easing-function/index.html]: it's weird, and I didn't like it, so I have just renamed them.

[the remark this answers is not in the document]
Comment (ericwbailey) [reply files/en-us/web/css/easing-function/index.html]: Boom.

**Review by ericwbailey — APPROVED** [review]

**Pull request merged**